packages/eslint-plugin-template/docs/rules/self-closing-tags.md

```diff
@@ -0,0 +1,237 @@
+<!--
+
+  DO NOT EDIT.
+
+  This markdown file was autogenerated using a mixture of the following files as the source of truth for its data:
+  - ../../src/rules/self-closing-tags.ts
+  - ../../tests/rules/self-closing-tags/cases.ts
+
+  In order to update this file, it is therefore those files which need to be updated, as well as potentially the generator script:
+  - ../../../../tools/scripts/generate-rule-docs.ts
+
+-->
+
+<br>
+
+# `@angular-eslint/template/self-closing-tags`
```

Comment:
To match existing conventions, please go for `prefer-self-closing-tags` as the rule name